atm.js

```diff
@@ -1,21 +1,84 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+var chkDeposit = $("#checking_deposit");
+var chkWithdrawal = $("#checking_withdraw");
+var savDeposit = $("#savings_deposit");
+var savWithdrawal = $("#savings_withdraw");
+var chkAmount = 0
+var savAmount = 0
 
-}
 
-function checkingWithdrawal(amount){
+// an eventListener for each button, each one a "click"
+chkDeposit.on("click", checkingDeposit);
+chkWithdrawal.on("click", checkingWithdrawal);
+savDeposit.on("click", savingsDeposit);
+savWithdrawal.on("click", savingsWithdrawal);
```

Comment:
nice! can you think of a way to combine the event listeners into one? maybe you can check the text of the button to decide which function to call